features/browsing_list_of_all_credit_memos.feature

```diff
@@ -24,6 +24,6 @@ Feature: Browsing list of all credit memos
         When I browse credit memos
         Then there should be 2 credit memos generated
         And 1st credit memo should be generated for order "#00000022", have total "$10.00" and date of being issued
-        And 1st credit memo should be issued in "United States" channel
+        And 1st credit memo should be issued in "WEB-US" channel
         And 2nd credit memo should be generated for order "#00000022", have total "$20.00" and date of being issued
-        And 2nd credit memo should be issued in "United States" channel
+        And 2nd credit memo should be issued in "WEB-US" channel
```

Comment:
I think we should still have channel name and just add color to it (see https://github.com/Sylius/RefundPlugin/issues/80#issuecomment-426626200)

Reply:
This will be different to invoices grid... but changed